packages/firebase_performance/lib/src/performance_attributes.dart

```diff
@@ -26,17 +26,23 @@ abstract class PerformanceAttributes {
   ///
   /// Updates the value of the attribute if the attribute already exists.
   /// The maximum number of attributes that can be added are
-  /// [maxCustomAttributes].
+  /// [maxCustomAttributes]. An attempt to add more than [maxCustomAttributes]
+  /// to this object, will return without adding the attribute.
```

Comment:
nit: remove comma

Also this might be worth writing a test for if we don't have one already.